### DISCUSSION TITLE
appeareance of tui-input-date

### DISCUSSION THREAD

**Discussion by vytautas-pranskunas-** [taiga-family/taiga-ui#9620] [Q&A] [answered]
is there a way to provide appeareance to tui-input-date? For now i do not see anything in options: https://github.com/taiga-family/taiga-ui/blob/d5422544caaa3e13d31e6ebd004bc8f272fd9cc8/projects/kit/tokens/input-date-options.ts#L28

**Comment by vytautas-pranskunas-** [comment]
Any update here? Feedback would be very helpful.

p.s. i have promoted Taiga-UI as our companys main UI library because of your, guys, responsivnes and willingness to help no matter that npm download is quite small comparing to prime-ng for instance :)

**Reply by waterplea** [reply, marked as answer]
I was on vacation :) `InputDate` follows the same DI tokens as others:
```html
    <tui-input-date
        [(ngModel)]="value"
        tuiTextfieldAppearance="secondary"
    >
        Choose a date
    </tui-input-date>
```
Don't forget `TuiTextfieldControllerModule`
